docs: add grok guide (#540)
fleek-platform/website: src/content/guides/grok-eliza-guide/index.md

1. Click on the C-3PO button, the form on the page will be filled instantly with the necessary details.

2. Go to the “Model providers” section and click to open the dropdown.

3. With the dropdown open, scroll to look for “Grok”. You should see the below:

![Model providers](./models.png)

4. Now with Grok selected, you need not change anything, but, you can add an avatar and maybe update the name based on your preferences.

![Avatar](./avatar.png)

5. With that done, scroll to the bottom and click on the “Continue to settings” button.

6. On this page you will fill in all required information. Some of the other fields have been already filled in and you wouldn’t need to edit those:

![Settings page](./settings.png)

7. After all that, you review your agent details and confirm that all the provided information is correct:

![Reviews page](./reviews.png)

**Note:** you can switch tabs to the “JSON” (Characterfile) one in case there is anything you would like to edit or modify.

8. After reviewing the information to confirm that all is well you then scroll to the bottom to click the “Deploy agent” button and you should see your agent go live in less than 30 seconds.

> Immediately after the deployment of your agent, you will see your agent already posting on X and you can even tag your agent to things for responses if need be. You can also go back to edit your Characterfile if you want to change anything about your agent:

Reviewer: ```suggestion
Great! Your agent will be live on X and ready to interact. Tag it for responses or head back to edit its Characterfile anytime:
```
